src/test/java/oss/fosslight/controller/RequestReviewIntegrationTest.java

```diff
@@ -0,0 +1,106 @@
+package oss.fosslight.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.transaction.annotation.Transactional;
+import oss.fosslight.domain.Project;
+
+import java.util.Map;
+import java.util.UUID;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@AutoConfigureMockMvc(addFilters = false)
+@SpringBootTest
+@Transactional
+public class RequestReviewIntegrationTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    private Project prj;
+
+    @BeforeEach
+    void createProject() throws Exception {
+        MockHttpServletResponse response = mockMvc.perform(post("/project/saveAjax")
+                .param("copy", "false")
+                .param("prjName", UUID.randomUUID().toString())
+                .param("prjVersion", "v1")
+                .param("noticeType", "10")
+                .param("distributionType", "10")
+                .param("comment", "<p>test</p>")
+                .param("osType", "100")
+                .param("priority", "30")
+                .param("statusRequestYn", "")
+                .param("listId", "")
+                .param("publicYn", "Y")
+                .param("networkServerType", "N")
+                .param("prjDivision", "999")
+                .param("prjUserId", "test2")
```

Comment:
[Question] Will a project be created even if I enter an ID that is not registered as a user? 
"test2" is not a user added by default, so I wonder if a project will be created.

Reply:
Project is created regardless of UserId.